File: codes/ChainingTable/Java/ChainingTable/Person.java
package ChainingTable;

public class Person {
    private int ssn;
    private String name;
    private String address;

    public Person() {
        this.ssn = 0;
        this.name = "";
        this.address = "";
    }

    public Person(int _ssn, String _name, String _address) {
        this.ssn = _ssn;
        this.name = _name;
        this.address = _address;
    }

    public int getSsn() {
        return this.ssn;
    }

    public void ShowPerInfo() {
        System.out.println("주민등록번호: "+this.ssn);
        System.out.println("이름: "+this.name);
        System.out.println("주소: "+this.address);
        System.out.println();
    }
}

File: codes/DoubleHashTable/Python/Person.py
class Person:
    def __init__(self, ssn=0, name="", address=""):
        self.ssn = ssn
        self.name = name
        self.address = address

    def ShowPerInfo(self):
        print("주민등록번호: {}".format(self.ssn))
        print("이름: {}".format(self.name))
        print("주소: {}".format(self.address))
        print()
